components/style/rule_tree/mod.rs

```diff
@@ -500,8 +500,8 @@ impl RuleNode {
     }
 
     fn iter_children(&self) -> RuleChildrenListIter {
-        // FIXME(emilio): Fiddle with memory orderings.
-        let first_child = self.first_child.load(Ordering::SeqCst);
+        // See next_sibling to see why relaxed is ok.
+        let first_child = self.first_child.load(Ordering::Relaxed);
```

Comment:
Given that next_sibling is currently |Acquire|, this needs to be |Acquire| as well.